README.md

```diff
@@ -0,0 +1,4 @@
+# BotLink
```

Comment:
decide on a casing. I suggest all lower-case, `blink`

Reply:
Ok, will address in PR #18